src/style.css

```diff
@@ -1 +1,63 @@
 /* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.nav__bar {
+  display: flex;
+  justify-content: space-between;
+  text-align: center;
+  align-items: center;
+  background-color: #fff;
+}
+
+img {
+  display: block;
+  margin: 10px 0 10px 50px;
+  width: 40px;
+  height: 40px;
+  text-decoration: none;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  margin-right: 50px;
+}
+
+.nav__link {
+  text-transform: uppercase;
+  text-decoration: none;
+  position: relative;
+  color: #000;
+  padding: 30px 0;
+  margin: 22px 20px 23px;
+}
+
+.nav__link[data-qa="hover"] {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```

Comment:
Move all repeatable colors to CSS variables.

Reply:
[More info](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)